conanfile.py

```diff
@@ -1,39 +1,44 @@
-from conans import ConanFile, CMake, tools
+from conans import ConanFile, CMake
 
 
 class LibgeohashConan(ConanFile):
     name = "Libgeohash"
-    version = "0.1"
-    license = "https://github.com/simplegeo/libgeohash/blob/master/LICENSE"
+    version = "2469d3c"
+    license = "BSD-3"
+    homepage = "https://github.com/simplegeo/libgeohash"
     url = "https://github.com/ebclark2/conan-libgeohash.git"
+    description = "A pure C implementation of the Geohash algorithm. http://geohash.org"
     settings = "os", "compiler", "build_type", "arch"
     options = {"fPIC": [True, False]}
     default_options = "fPIC=False"
```

Comment:
I usually default it to True, any special reason?

Reply:
no reason, it was like that, but enabling it by default